### ISSUE TITLE
unable to reload

### ISSUE THREAD

**Issue by anurat** [bangnokia/laravel-serve-livereload#9]
Hello,

Thank you so much for making this package.  
I tried vscode live server before but it didn't work.
So I tried this package on Laravel ^8.12, I installed this package with version ^1.1
and it's on Windows 10, powershell prompt.

and then run `php artisan serve` which showed websockets server started

```
Starting Laravel development server: http://127.0.0.1:8000
[Tue Mar 23 11:55:03 2021] PHP 7.4.11 Development Server (http://127.0.0.1:8000) started
Websockets server started: ws://127.0.0.1:1322
```

I write a simple test on `routes/web.php`

```php
Route::get('/test', function () {
    return 'hello';
});

```
and tried to change return value and save but my browser (localhost:8000/test) still doesn't automatically change.  
I wonder what I missed

Best Regards,

**Comment by bangnokia** [comment]
Can you try reload your browser and view the page source? If you see something like this

```javascript
(new WebSocket('ws://127.0.0.1:1322')).onmessage = function (message) {
        if (message.data === 'reload') {
            window.location.reload(true);
        }
    };
```

that's mean it works, other wise, may be you should try to clear the cache first by `php artisan cache:clear`.

**Comment by anurat** [comment]
Thanks for the reply!  
I tried `php artisan cache:clear` `php artisan optimize:clear`,  `composer dump-autoload`

but nothing on page source, only "hello" string so far.

**Comment by anurat** [comment]
Now I tried your package with a fresh Laravel project and it works without a problem!

I guess my current project has conflicting packages with this but will need more investigation.  
Anyway, Thanks for your help!

**Comment by bangnokia** [comment]
You are welcome, i don't use windows so i have no idea about your problem. But this package is simple, let me explain how it works in case you  want to investigate why.

You just  watch at class `ServeWebSocketsCommand` it starts a websocket server, and run a file watcher `Watcher` for every time file changed, it's send a message reload to the websocket listeners. 
The watcher has run interval every 0.5 second and put a value to Cache `Cache::put('serve_websockets_running', true, 5);` to helping detection if the websocket server is running.
There is a middleware `InjectScriptsMiddleware` read the value from cache and try to inject some javascript into the response if the websocket server is running.

I think there is a problem some where when the middleware can not inject the script into your application response.

Thank you  for using this silly package :D

**Comment by anurat** [comment]
It turns out that the interfering package is **Laravel Sanctum**.  
When I add Laravel Sanctum package, the websockets script isn't present.  
And when I remove Laravel Sanctum, I got the script back again.

I can't look into this any further but hope this helps someone :)

And @bangnokia if you happens to find a way to make your package work with Sanctum, that'd be great!

**Comment by bangnokia** [comment]
@anurat thank you so much!
fixed at version 1.1.3
